website/docs/patterns/data-visualization/partials/code.md

```diff
@@ -0,0 +1,9 @@
+## How to use Carbon charts components
+
+We recommend using the Vanilla JavaScript version of IBM’s [Carbon Chart components](https://carbondesignsystem.com/data-visualization/chart-types/) as a basis for creating your own Ember components.
```

Comment:
> as a basis for creating your own Ember components

We don't know if they will create components (as wrappers of the Carbon Charts, like you did in your spike) or use directly the charts in template/controller pages, or potentially use other tools/approached provided by the Ember framework (an engine? a service? 🤷‍♂️). We should probably keep this more generic.

Reply:
I updated the content to make the recommendation more general. I also added some example code for creating an Ember component.

https://hds-website-git-hds-data-viz-web-docs-hashicorp.vercel.app/patterns/data-visualization?tab=code